UIKitDZ/UIKitDZ/Model/Source.swift

```diff
@@ -0,0 +1,99 @@
+// Source.swift
+// Copyright © RoadMap. All rights reserved.
+
+/// Структура для постов
```

Comment:
```suggestion
/// Структура для постов
```
Не уверен, что это структура только для постов.